### ISSUE TITLE
Curation filter on author patterns page returns a 404

### ISSUE THREAD

**Issue by enrico-sorcinelli** [WordPress/pattern-directory#706]
**Describe the bug**
By applying Curation filter on author patterns page return a 404 HTTP status code.

**To Reproduce**
- Go to a author patterns page (e.g: https://wordpress.org/patterns/author/colorful-tones/, https://wordpress.org/patterns/author/deepakrohilla/).
- Select _Curated_ filter and press _Apply_.
- A 404 page is returned.

**Expected behavior**
A result page with HTTP 200 and message like "No results found" IMHO could be more appropriate.
PS: If it doesn't make sense to apply this filter to community authors, then it might be disabled/hidden at all on that case.

**Comment by ryelle** [comment]
>  PS: If it doesn't make sense to apply this filter to community authors, then it might be disabled/hidden at all on that case.

Yeah, I think in reality, only patterns by the wordpressdotorg user are going to be curated, so we can hide this on all author archives.
